components/navbar/navbar.scss

```diff
@@ -1,142 +1,136 @@
-#burger {
```

Comment:
If certain pixel values in this file need to match I will turn them into variables.

e.g., `28px`, `12.5px` (should be changed to an integer number)

Reply:
okay, will do 👍